### DISCUSSION TITLE
Student Developer Pack approved but no benefit available yet

### DISCUSSION THREAD

**Discussion by Tostiffent** [community/community#126943] [GitHub Education] [answered]
### Hi everyone,

Hello, i have submitted my application for GitHub education pack on 30th June 2024 and was approved within a few minutes but i have still not received my benefits or any email regarding the same. 

Please guide me on what steps can i take next because i need this pack for a project I've been working on.

![image](https://github.com/community/community/assets/99608933/8ca774e5-34a4-48c6-9b92-8a1aa63898b1)

**Comment by ExpiredWater** [comment, marked as answer]
Hi @Tostiffent ,

If your application request has a green badge and a message saying "Congrats! Your request for @Tostiffent was approved and your benefits will be available soon." it means that you have to wait, that's why your benefits aren't yet available.

![image](https://github.com/community/community/assets/24321664/1ff82946-ff16-4bdf-b977-cd8c81583cf3)

If your application request has a purple badge and a message saying "Congrats! Your request for @Tostiffent was approved and your benefits are now available." it means that your benefits should be available, so if you can't access them you should contact [GitHub Support](https://support.github.com/request/education)

![image](https://github.com/community/community/assets/24321664/ff3b529e-32d3-437d-8d60-82c4734eafa6)
